src/JustEat.StatsD/IpTransport.cs

```diff
@@ -15,21 +15,14 @@ public IpTransport(IPEndPointSource endPointSource)
         }
 
         public void Send(string metric)
-        {
-            Send(new[] {metric});
-        }
-
-        public void Send(IEnumerable<string> metrics)
         {
             var endpoint = _endpointSource.GetEndpoint();
-            var packets = metrics.ToMaximumBytePackets();
+            var bytes = Encoding.UTF8.GetBytes(metric);
 
             using (var socket = MakeIpDatagramSocket())
             {
-                foreach (var packet in packets)
-                {
-                    socket.SendTo(packet, endpoint);
-                }
+                socket.Connect(endpoint);
```

Comment:
Is this call to `Connect()` required given it wasn't there before?

Reply:
Rather than use `SendTo`, you can use `Connect` & `Send` separately, this means we can take advantage of the overloads on `Send` that support `Span<T>` and family

Reply:
We have changed `SendTo` to `Connect; Send` which we think is equivalent. Can change back for now if it's not.